supporter-product-data/src/main/scala/com/gu/lambdas/AddSupporterRatePlanItemToQueueLambda.scala

```diff
@@ -98,60 +105,17 @@ object UpdateDynamoLambda extends StrictLogging {
       logger.info(
         s"Continuing processing with batch of ${batch.length} - time remaining: ${timeOutCheck.timeRemainingMillis / 1000} seconds, buffer: ${timeoutBufferInMillis / 1000} seconds",
       )
-
-      Await.result(writeBatch(batch, dynamoDBService, alarmService), 120.seconds)
+      try {
```

Comment:
if you use a scala Try instead, it will only catch NonFatal
presume this is only if the whole batch fails, e.g. credential issue, rather than individual items

Reply:
I'd be tempted to make sendBatch return a Try or similar, to minimise the reach of exceptions through the code.

Reply:
I've consolidated this into the SqsService as there were two separate bits of code both handling errors from sqs